packages/react-client/src/ReactFlightClientStream.js

```diff
@@ -36,33 +34,20 @@ function processFullRow(response: Response, row: string): void {
   if (row === '') {
     return;
   }
-  const tag = row[0];
+  const colon = row.indexOf(':', 0);
```

Comment:
are you leaving the position argument in place to be emphatic that it searches the whole string?

Reply:
No reason really other than why not be explicit and I’m not sure this will last. Eg might be continuing from a partial string when we optimize this a bit.